spec/default_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe 'yum-cron::default' do
+  [CENTOS_6_OPTS, CENTOS_7_OPTS].each do |pltfrm|
+    describe "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(pltfrm).converge(described_recipe)
+      end
+
+      it do
+        expect(chef_run).to install_package('yum-cron')
+      end
+
+      case pltfrm
+      when CENTOS_6_OPTS
+        it do
+          expect(chef_run).to create_template('/etc/sysconfig/yum-cron')
+        end
+      when CENTOS_7_OPTS
+        it do
+          expect(chef_run).to create_template('/etc/yum/yum-cron.conf')
```

Comment:
Can you please verify the file renders with the correct data? (I can show you how to do that).